pkg/api/controller.go

```diff
@@ -58,7 +60,7 @@ func (c *Controller) Run() error {
 
 	engine := mux.NewRouter()
 	engine.Use(DefaultHeaders(),
```

Comment:
Why don't we do this?
if c.Audit {
engine.Use(AuditLogger)
}

Reply:
Do you mean having 2 separate middleware functions?

Reply:
Yes, wouldn't that be cleaner?
Sorry it is taking a little longer with this PR, but let's fix it cleanly.